sdp.go

```diff
@@ -55,6 +57,16 @@ func trackDetailsFromSDP(log logging.LeveledLogger, s *sdp.SessionDescription) m
 				}
 			}
 
+			// Handle a=msid for Unified plan
+			// TODO make "msid" a constant
```

Comment:
https://github.com/pion/sdp/blob/99a4a121c465b7ab633505dcea9a96e2ccca9066/jsep.go#L11-L22

Reply:
Thanks! I created a pull request in `pion/sdp`: https://github.com/pion/sdp/pull/47